docs/ua/requirements/state-of-the-art.md

```diff
@@ -2,33 +2,144 @@
+| Performance    |                                     |                                       |                            |                                                   |                          |                                       |                                     |              |               |                                       |
+|                | Швидкість                           | `4`                                   | `5`                        | `5`                                               | `5`                      | `5`                                   | `5`                                 | `4`          | `5`           | `5`                                   |
+|                | Оптимізація                         | `3`                                   | `5`                        | `5`                                               | `5`                      | `5`                                   | `5`                                 | `4`          | `5`           | `5`                                   |
+| Supportability |                                     |                                       |                            |                                                   |                          |                                       |                                     |              |               |                                       |
+|                | Служба підтримки                    | `5`                                   | `5`                        | `5`                                               | `5`                      | `5`                                   | `5`                                 | `5`          | `5`           | `5`                                   |
+|                | FAQ                                 | `5`                                   | `5`                        | `5`                                               | `5`                      | `5`                                   | `5`                                 | `5`          | `5`           | `5`                                   |
 
 ## Висновки
 
-_[Робляться висновки щодо доцільності розробки нової або модифікації існуючої інформаційної системи, необхідності та способів інтеграції з системами(сервісами) третіх сторін, тощо.]_
+Провівши аналіз предметної області, було виявилено необхідність у створенні нової, досконалішої системи. Тому наша група вирішила розробити нову систему аналізу медіа-контенту. При розробці ми візьмемо до уваги недоліки, що були виявлені у інших подібних системах.
 
 ## Посилання
 
-_[Розділ містить повний список всіх документів, про які згадується.]_
+**\*Аналіз** (2023) Wikipedia. Wikimedia Foundation. Available at: [Link](https://en.wikipedia.org/wiki/Analysis)\*
```

Comment:
delete ```*``` at rhe begining and ending, remove dates and  ```Available at```(just link), replace unordered list to ordered one

Reply:
what u mean by replace unordered list to ordered one?